### PR TITLE
upload user's avatar (profile image)

### PR DESCRIPTION
**Pull request by Rupinderwadali**
Add new field in devise registration form for uploading user profile photo

### CODE REVIEW

File: app/views/devise/registrations/edit.html.erb [hunk 1]
```diff
@@ -104,8 +104,19 @@
                             <span class="help-inline">(If you don't want to change your username or password then leave this field blank.)</span>
                         </div>
                     </div>
-
-                <% end %>
+	     <div class="Form-group">
```
Comment (valeriecodes) [inline app/views/devise/registrations/edit.html.erb]: Downcase div name

File: app/views/devise/registrations/edit.html.erb [hunk 2]
```diff
@@ -104,8 +104,19 @@
...
+                   	      <%= f.label :Profile_image %><br>
```
Comment (valeriecodes) [inline app/views/devise/registrations/edit.html.erb]: Downcase label sym

File: app/views/devise/registrations/new.html.erb [hunk 3]
```diff
@@ -55,8 +55,14 @@
             <%= f.label :password_confirmation %>
             <%= f.password_field :password_confirmation, class: "Form-control" %>
         </div>
+ <div class="Form-group">.
```
Comment (valeriecodes) [inline app/views/devise/registrations/new.html.erb]: Downcase div name

**Comment by Rupinderwadali** [comment]
Okay. Will change it soon :)

**Comment by adityathiru** [comment]
Tested PR. Encountered the following issues: 
1. Unfriendly error messages on uploading avatar : 
Error: Avatar translation missing: en.errors.messages.mini_magick_processing_error
(To rectify this, imagemagick needs to be installed on your PC.)

2. User Image can be used in the top left corner dropdown (Enhancement)
To update this (tested from my end) :
change line 5 of app/views/shared/_header.html.erb from ```<%= image_tag ("user.png")%>``` to ```<%= image_tag(current_user.avatar.url(:thumb), unless: current_user.avatar.nil?) %>```

3. Dropdown button not functioning.

**Pull request closed without merging**